app/workers/direct_deposit_email_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DirectDepositEmailJob
+  include Sidekiq::Worker
+  sidekiq_options expires_in: 1.day
```

Comment:
My thinking here is that if the email isn't successfully sent within 24 hours, don't keep trying.
If I updated my info, but received a confirmation email 4 days later, I might not associate it with my update 4 days earlier.  I'd be confused and worried that a nefarious actor had updated it after me.